data/Odohofre.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Bright Odohofre",
+  "type": "personal",
+  "displayStatsPublic": true,
```

Comment:
We recently have removed the public stats feature, so `displayStatsPublic` property is no longer needed. 
Please remove this field from your json.

```suggestion
```